### PR TITLE
adding script for 100bp coverage stats and unique isoform searches

### PR DESCRIPTION
**Pull request by evanbiederstedt**

### CODE REVIEW

File: scripts/coverage_stats.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# -*- coding: UTF-8 -*-
```
Comment (ame9yu) [inline scripts/coverage_stats.py:2]: Please add some documentation for each script in a readme file.

Comment (evanbiederstedt) [inline scripts/coverage_stats.py:2]: Adding a python shebang here would be useful

File: scripts/find_unique_isoform.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# -*- coding: UTF-8 -*-
```
Comment (evanbiederstedt) [inline scripts/find_unique_isoform.py:2]: Shebang here too please 
`#!/usr/bin/env python3`

**Comment by evanbiederstedt** [comment]
@BidaGu this was originally your PR I think

Should we merge this or close?

**Comment by BidaGu** [comment]
> @BidaGu this was originally your PR I think
> 
> Should we merge this or close?

I think we can merge. I have addressed all the issues here with the most recent commit.

**Pull request merged**